### PR TITLE
[WIP] Add caption to Emoji file

### PR DESCRIPTION
**Pull request by ServatDjoo**
GitHub Training on Tuesday November 7

### CODE REVIEW

File: _posts/0000-01-07-ServatDjoo.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![homer-hide](https://cloud.githubusercontent.com/assets/16547949/25400902/0a553f14-29c2-11e7-9a40-8d14136b4fa3.jpg)
```
Comment (githubteacher) [inline _posts/0000-01-07-ServatDjoo.md:6]: We got the same meme!

Meme bros!

File: _posts/0000-01-07-ServatDjoo.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
...
 {: .center}
 
-Caption here
+Good looking guy, just need a shave!
```
Comment (githubteacher) [inline _posts/0000-01-07-ServatDjoo.md:9]: 🌳

**Review by githubteacher — COMMENTED** [review]
This looks great!

**Comment by githubteacher** [comment]
General conversation comment goes here.

**Review by githubteacher — APPROVED** [review]

**Comment by ServatDjoo** [comment]
Mergre Pull Request

**Pull request merged**